gateway/modules/crud/bolt/create.go

```diff
@@ -0,0 +1,100 @@
+package bolt
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/boltdb/bolt"
+	"github.com/sirupsen/logrus"
+
+	"github.com/spaceuptech/space-cloud/gateway/model"
+	"github.com/spaceuptech/space-cloud/gateway/utils"
+)
+
+// Create inserts a document (or multiple when op is "all") into the database
+func (b *Bolt) Create(ctx context.Context, project, col string, req *model.CreateRequest) (int64, error) {
```

Comment:
You have seperated out the implementation of `all` and `one` operations. I would suggest you only implement the `all` operation. Convert the object to an array if it was a `one` operation. Let's keep things DRY.